lesson06/task3/server/main.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"time"
+)
+
+var name string
+var address string
+
+func handler(w http.ResponseWriter, r *http.Request) {
+
+	if r.URL.Path != "/" {
+		http.Error(w, "404 not found.", http.StatusNotFound)
+		return
+	}
+
+	switch r.Method {
+	case "GET":
+
+		// GET return HTML page
+		http.ServeFile(w, r, "form.html")
```

Comment:
You don't have the `form.html` in this directory or anywhere in the project. You need to add it and provide a correct path if you want your server to be able to serve it. Right now the server returns 404.

Reply:
Sorry. It is my mistake. I didn't add task1, task 2 as well as form.html. Now I fix it. As well, I need time to fix task3.